Aztec/Classes/TextKit/TextView.swift

```diff
@@ -611,11 +613,13 @@ open class TextView: UITextView {
         formatter.toggleAttribute(inTextView: self, atRange: range)
     }
 
-    /// Refresh Lists attributes when text is deleted in the specified range
+
+    /// Toggles the Blockquote Style, whenever the deleted text is at the Location 0, which is 
+    /// also known as the "Beginning of the text"
```

Comment:
Also consider changing to "...whenever the deleted text is at location 0."  The "also known as the beginning of the text" part should be pretty clear anyway.